README.md

```diff
@@ -144,6 +161,104 @@ module.exports = ({ renderJsBundles, renderCssLinks, production }) => {
 };
 ```
 
+## Deployment environments
+
+### Env vars
+
+Underreact allows you to inject environment variables **during the build time**. You can set them up by creating a `.env` file at the root of your project:
+
+```
+#.env
+SERVER=https://example.com
+```
+
+You can then use them in your codebase:
+
+```javascript
+const url = process.env.SERVER;
+fetch(`${url}/data`);
+```
+
+Any env variable that you use in your app must be declared in the `.env` file. This is required to prevent unintentional leaking of env vars in your javascript bundle. 
+
+If you want to pass an environment variable in the CLI, make sure it is declared in your `.env` file first and use it like:
+
+```
+SERVER=https://foobar.com npx underreact start
```

Comment:
For this example could you add the corresponding `.env` file? I'm wondering, for example, if I need to have a line saying `SERVER=SERVER`, or if my `SERVER=https://one.two.com` line will be overridden?